Added reload action to the opened files context menu.

martonmiklos opened the pull request:
When using an external CAM software it is common use case that the modified G code needs to be reloaded. 
I have added an action to the context menu to make this action more easier.

martonmiklos commented:
I have added reload file to the context menu:
![kep](https://cloud.githubusercontent.com/assets/1609182/21083442/1c122ab8-bff1-11e6-85ea-0efd5be226ea.png)
And recent projects to the file menu:
![kep](https://cloud.githubusercontent.com/assets/1609182/21083436/07fc738a-bff1-11e6-8d46-2dd14567453c.png)

jcoffland commented on src/camotics/qt/QtWin.cpp:
```diff
@@ -1541,6 +1598,11 @@ void QtWin::animate() {
 }
 
+void QtWin::openRecentProjectsSlot(const QString path)
+{
```
This brace should be on the line above.

jcoffland commented on src/camotics/qt/QtWin.cpp:
```diff
@@ -818,6 +826,55 @@ void QtWin::openProject(const string &_filename) {
+    settings.setArrayIndex(size);
+    settings.setValue("fileName", QString::fromStdString(filename));
+    settings.endArray();
+  }
```
I'd like to see this moved to it's own function.

jcoffland commented on qt/camotics.ui:
```diff
@@ -3042,6 +3049,11 @@
   </action>
+  <action name="actionFoobar">
+   <property name="text">
+    <string>Foobar</string>
```
What is the purpose of this property?

martonmiklos commented:
I have added an important feature to the recent list: limited the history size to 20 items. Implementing this feature had eliminated the first review.

I have fixed the indentation issue, also removed the foobar action from the ui file. It was just used for testing, thanks for spotting it!

jcoffland commented:
This looks great.  Thanks.  I'm going to commit a few minor formatting tweaks which you might want to pull before doing any further work.

martonmiklos commented:
Cool! Thanks for the review and the heads up!
